README.md

```diff
@@ -14,68 +14,63 @@
-The app's icon can be changed to a seemingly harmless one, such as a calcultor or a camera. This means that an individual searching the device will not see the Tella name and icon, and instead see an app that doesn't raise suspicions.
-WARNING: the name and icon of Tella will remain visible in the Android settings; this means that this camouflage will not protect against an individual actively looking for Tella or conducting an in-depth analysis of the device. This is a good protection against superficial searches of the user's device, avoiding to raise suspicions if an individual is quickly searching for incriminating evidence.
+## Detailed list of features <a id="features"></a>
 
-### Data collection
-Users working with an organization can upload photos, videos, and audio files, or fill and submit forms, to document events they are witnessing. The organization decides which method is most appropriate to their needs and capacity: receiving the files through a third-party app like WhatsApp or Signal; or using forms get ask questions about the specific data they need. 
+A detailed list of features for both Tella Android and iOS can be found [on the documentation](https://tella-app.org/features).
 
-Previous Tella deployments have ranged between 10 and 2,000 users. 
-Tella includes an "Offline Mode" for users who are collecting data in remote areas with limited or no internet connection. When Offline Mode is activated, all data is saved on the app and and users can submit it easily when they reach an internet connection.
 
-### Wipe data
-A Quick Delete button allows users, in a just a few seconds, to delete sensitive data within Tella. The button can also be set to delete the app itself.
+## How to get Tella and start using it? <a id="use-tella"></a>
 
-### Verification
-In the Tella settings, users can activate "Verification Mode". When activated, every time a user takes a photo or a video, or records audio, Tella automatically captures metadata about the file. This metadata can be used to corroborate evidence, cross check with other facts known about the event or about the area where it was captured.
+### Tella for iPhone
+Tella for iOS can downloaded [from the App Store](https://apps.apple.com/us/app/tella-document-protect/id1598152580).
 
-## How to get Tella and start using it? <a id="use-tella"></a>
+### Get started on Tella iOS
+A get started guide for Tella iOS is available [here](https://tella-app.org/get-started-ios).
 
-Tella is publicly available only on Android. You can [download it](https://play.google.com/store/apps/details?id=org.hzontal.tella) from the Google Play Store or [get the APK from here](https://web.tresorit.com/l/7737s#WKoGVN82C0lQK8KSL30CuA) to install manually on your device.
 
 ## Tech & frameworks used <a id="tech-used"></a>
 - [Secure Enclave](https://support.apple.com/guide/security/secure-enclave-overview-sec59b0b31ff/web) for security
```

Comment:
can we add the third libraries used : 
 * [Mantis] For edit image feature (https://github.com/guoyingtao/Mantis) 
 * [GoogleSignIn] and [GoogleAPIClientForREST/Drive] to implement google drive connection (https://github.com/google/GoogleSignIn-iOS) , (https://github.com/google/google-api-objectivec-client-for-rest)
 * [SQLCipher] for encrypted DataBase (https://github.com/sqlcipher/sqlcipher)
 * [NextcloudKit] a package added to implement Nextcloud connection (https://github.com/nextcloud/NextcloudKit)

Reply:
Maybe the PDFKit is an internal package, I think there is no need to add it to this file

Reply:
great! I just applied these changes and also linked to the description of these features :)

The link for nexcloud won't work because I didn't wrote it yet, but it will be in that url